packages/kakoune/build.sh

```diff
@@ -0,0 +1,18 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/mawww/kakoune
+TERMUX_PKG_DESCRIPTION="Code editor heavily inspired by Vim"
+TERMUX_PKG_VERSION=2018.04.13
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_SRCURL=https://github.com/mawww/kakoune/archive/v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=dfbaf87479ae30fd87426ae1b5f6cbe4382d6fe17b64a0c58d9475bf038e50dc
+TERMUX_PKG_DEPENDS="ncurses"
+TERMUX_PKG_BUILD_IN_SRC=yes
+
+termux_step_make(){
+   cd src
+   make TERMUX_PREFIX="${TERMUX_PREFIX} debug=no"
```

Comment:
You could use `TERMUX_PKG_EXTRA_MAKE_ARGS=" -C src"` and skip overriding termux_step_make (should work in theory), at least if `debug=no` is the default option.